ledger/apply/asset.go

```diff
@@ -231,7 +231,7 @@ func putIn(balances Balances, addr basics.Address, asset basics.AssetIndex, amou
 		return err
 	}
 	if !ok {
-		return fmt.Errorf("asset %v missing from %v", asset, addr)
+		return fmt.Errorf("receiver error: must optin, asset %v missing from %v", asset, addr)
```

Comment:
nit: why receiver error?

Reply:
Honestly it was just what was suggested in #3775